src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design/Internal/CommonOptions.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design.Internal
+{
+    public class CommonOptions
+    {
+        public CommandOption OutputPathOption { get; private set; }
+
+        public CommandArgument ProjectArgument { get; private set; }
+
+        public CommandOption ConfigureCompilationType { get; private set; }
+
+        public CommandOption ContentRootOption { get; private set; }
+
+        public CommandOption GeneratePdbOption { get; private set; }
+
+        public void Configure(CommandLineApplication app)
+        {
+            app.Description = "Precompiles an application.";
+            app.HelpOption("-?|-h|--help");
+
+            ProjectArgument = app.Argument(
+                "project",
+                "The path to the project (project folder or project.json) with precompilation.");
+
+            OutputPathOption = app.Option(
+                "-o|--output",
```

Comment:
This might be changed by https://github.com/aspnet/Mvc/pull/5087/files#r73390069